.github/workflows/stack-ci.yml

```diff
@@ -76,9 +76,9 @@ jobs:
 
       # Installs the recent protoc, for now unpinned to gain some experience
       - name: Install Protoc
-        uses: arduino/setup-protoc@v1
```

Comment:
Mayhaps v1 doesn't support downloading v28.0.  Updated to the latest v3.

Reply:
This indeed seems to have been the case.  `protoc` installs successfully when testing in a pull request within my own fork: https://github.com/chungyc/proto-lens/pull/10